src/zulqer/narcissisticNumbers/index.ts

```diff
@@ -0,0 +1,22 @@
+function isNarcissisticNumber(num: number): boolean {
```

Comment:
Since there is a different pull request exist for `narcissisticNumbers`, please remove this file from this pull request. As both are independent features there is a chance one pull request will be promoted before other.